assets/base.css

```diff
@@ -2246,22 +2250,43 @@ product-info .loading-overlay:not(.hidden) ~ *,
   line-height: calc(1 + 0.1 / var(--font-body-scale));
 }
 
-/* section-announcement-bar */
-.announcement-bar {
+/* utility-bar */
+.utility-bar {
+  height: 100%;
+}
+
+.utility-bar--bottom-border {
   border-bottom: 0.1rem solid rgba(var(--color-foreground), 0.08);
+}
+
+.announcement-bar,
+.announcement-bar__announcement {
   color: rgb(var(--color-foreground));
+  width: 100%;
```

Comment:
Not a huge fan of how far the arrows are from the content: [video](https://screenshot.click/31-12-sougk-1mo2o.mp4)

Reply:
Hey @YoannJailin this needs your attention. Thanks.

Reply:
Hello @eugenekasimov @ludoboludo i think it could be good to make the arrow icons more centered so the user can see them more easily / he won't have to go too far to hover it on large screens. 

Let's just add the additional options for now (selectors, social media icons) that will shrink the Announcements slider to the middle including the arrows. We can still make that narrow version of the slider default later, even if there's no other options activated.